src/filesys.cpp

```diff
@@ -698,22 +699,37 @@ bool safeWriteToFile(const std::string &path, const std::string &content)
 		return false;
 	}
 
+	bool rename_success = false;
+
 	// Move the finished temporary file over the real file
 #ifdef _WIN32
+	// When creating the file, it can cause Windows Search indexer, virus scanners and other apps
+	// to query the file. This can make the move file call below fail.
+	// We retry up to 5 times, with a 1ms sleep between, before we consider the whole operation failed
+	int number_attempts = 0;
+	while (!rename_success && number_attempts < 5)
+	{
```

Comment:
This brace should be on the previous line.  Also, consider making this a function (`fs::Rename()`?) considering how complex it is.